🐞 setPluginRegistrantCallback cannot find symbol Java

I'm getting this error, am I missing something?
```
Application.java:12: error: cannot find symbol
    WorkmanagerPlugin.setPluginRegistrantCallback(this);
                     ^
  symbol:   method setPluginRegistrantCallback(Application)
  location: class WorkmanagerPlugin
1 error

FAILURE: Build failed with an exception.
```

Application.java 

```
package`package_name`;

import be.tramckrijte.workmanager.WorkmanagerPlugin;
import io.flutter.app.FlutterApplication;
import io.flutter.plugin.common.PluginRegistry;
import io.flutter.plugins.GeneratedPluginRegistrant;

public class Application extends FlutterApplication implements PluginRegistry.PluginRegistrantCallback {
  @Override
  public void onCreate() {
    super.onCreate();
    WorkmanagerPlugin.setPluginRegistrantCallback(this);
  }

  @Override
  public void registerWith(PluginRegistry registry) {
    GeneratedPluginRegistrant.registerWith(registry);
  }
}
```
I've added the custom application to android manifest.

I have the dependency: workmanager: ^0.0.11

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+# 0.0.12
+
+* expose `setPluginRegistrantCallback` to the old rusty Java people 
+
 # 0.0.11
 
 * Better README and docs

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@ This is especially useful to run periodic tasks, such as fetching remote data on
 
 ```yaml
 dependencies:
-  workmanager: ^0.0.11
+  workmanager: ^0.0.12
 ```
 ```shell script
 flutter pub get

--- a/android/src/main/kotlin/be/tramckrijte/workmanager/WorkmanagerPlugin.kt
+++ b/android/src/main/kotlin/be/tramckrijte/workmanager/WorkmanagerPlugin.kt
@@ -2,9 +2,9 @@ package be.tramckrijte.workmanager
 
 import io.flutter.plugin.common.MethodCall
 import io.flutter.plugin.common.MethodChannel
-import io.flutter.plugin.common.PluginRegistry
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler
 import io.flutter.plugin.common.MethodChannel.Result
+import io.flutter.plugin.common.PluginRegistry
 import io.flutter.plugin.common.PluginRegistry.Registrar
 
 class WorkmanagerPlugin(private val workmanagerCallHandler: WorkmanagerCallHandler) : MethodCallHandler {
@@ -18,6 +18,7 @@ class WorkmanagerPlugin(private val workmanagerCallHandler: WorkmanagerCallHandl
             channel.setMethodCallHandler(WorkmanagerPlugin(WorkmanagerCallHandler(registrar.activeContext())))
         }
 
+        @JvmStatic
         fun setPluginRegistrantCallback(pluginRegistryCallback: PluginRegistry.PluginRegistrantCallback) {
             WorkmanagerPlugin.pluginRegistryCallback = pluginRegistryCallback
         }

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
 name: workmanager
 description: Flutter Workmanager. This plugin allows you to schedule background work on Android and iOS.
-version: 0.0.11
+version: 0.0.12
 author: Tim Rijckaert <tim.rijckaert@gmail.com>, Jeremie Vincke <jeremie@movify.com>
 homepage: https://github.com/vrtdev/flutter_workmanager
 

